hazelcast/src/main/java/com/hazelcast/cp/internal/CPSubsystemImpl.java

```diff
@@ -54,6 +54,7 @@
  * Provides access to CP Subsystem utilities
  */
 public class CPSubsystemImpl implements CPSubsystem {
+    public static final String CPMAP_LICENSE_MESASGE = "CPMap is not included in your license";
```

Comment:
CPMAP_LICENSE_MESASGE --> CPMAP_LICENSE_MESSAGE

Reply:
Ah, annoying. That will teach me for doing the change while tired. Will fix, otherwise it will annoy me each time I see it!